tests/cache/sorted_set.rs

```diff
@@ -191,15 +191,15 @@ pub mod sorted_set_fetch_by_score {
     }
 }
 
-pub mod sorted_set_get_rank {}
+mod sorted_set_get_rank {}
 
-pub mod sorted_set_get_score {}
+mod sorted_set_get_score {}
 
-pub mod sorted_set_get_scores {}
+mod sorted_set_get_scores {}
 
-pub mod sorted_set_increment_score {}
+mod sorted_set_increment_score {}
 
-pub mod sorted_set_remove_element {}
+mod sorted_set_remove_element {}
```

Comment:
does your `-t` stuff still work with the nested directory structure? I had tried to do this a few weeks ago and it seemed like rust really didn't want the nested directories and was fighting me all the way until i gave up

Reply:
Yes it seems that for integration tests this is OK. For doctests it needs to be visible at the crate level.